features/lint.feature

```diff
@@ -0,0 +1,36 @@
+Feature: Lint all starters
```

Comment:
Why do we need e2e tests for this?

Reply:
We can't run linters normally on the repo because of all the `{{ }}` in the file paths, so next best thing was to run `kedro lint` on them - we have something like that in kedro too, for brand new empty project. I don't know if it's _needed_, but it's _nice to have_ them all nice and clean so that when someone creates a project from a starter, `kedro lint` passes from the beginning. 😅

Reply:
Makes sense! 😃